action-creators/team-action-creators.js

```
@@ -88,6 +88,15 @@ export const askToJoinTeam = (team: Object, user: Object): ThunkType => {
     return thunk;
 };
 
+export const removeTeamRequest = (teamId: string, user: Object): ThunkType => {
+    function thunk() {
+        firebaseDataLayer.removeTeamRequest(teamId, user);
+    }
+
+    thunk.interceptOnOffline = true;
```

I need to learn more about how our offline behavior functions, but from the looks of this it appears to me that the user's removal request is just silently discarded. That wouldn't surprise me - and I wouldn't be surprised if that pattern exists elsewhere throughout the app. But @mpettit if you also think that is true, would you add a `//TODO` comment here and a low-priority issue in the queue to return to this? Much appreciated.

---

I do think that is true.  I've added issue #247 to investigate/address this